packages/kbn-esql-validation-autocomplete/src/definitions/functions.ts

```diff
@@ -481,14 +480,9 @@ export const evalFunctionsDefinitions: FunctionDefinition[] = [
     signatures: [
       {
         params: [{ name: 'field', type: 'string' }],
-        returnType: 'version',
+        returnType: 'string',
         examples: [`from index | EVAL version = to_version(stringField)`],
       },
-      {
-        params: [{ name: 'field', type: 'version' }],
-        returnType: 'version',
-        examples: [`from index | EVAL version = to_version(versionField)`],
-      },
     ],
```

Comment:
We aren't using Elasticsearch types yet (look at https://github.com/elastic/kibana/issues/174710). So, `version` can just be `string` and that makes everything work again.